### ISSUE TITLE
pdf upscale not working

### ISSUE THREAD

**Issue by nvmend** [Tenpi/Waifu2x-GUI#18]
pdf not working, after finish it result in a pdf file of 15kb and says that cant open it

**Comment by Tenpi** [comment]
Does it output the pages and upscale each page?

**Comment by puff-dayo** [comment]
OS: Windows 11 21H2 x64
Waifu2x-GUI Version: 0.4.7 (latest)

I used a mixed image and text layout pdf file for testing and reproduced the same problem.
Attached are the input files and all the files output by Waifu2x-GUI.
[Test PDF file and output.zip](https://github.com/Tenpi/Waifu2x-GUI/files/14349847/Test.PDF.file.and.output.zip)

Thank you for developing, and I'm happy to help with any further testing.

**Comment by Tenpi** [comment]
update to 0.4.8 and tell me if that works

**Comment by puff-dayo** [comment]
![Image](https://github.com/Tenpi/Waifu2x-GUI/assets/84665734/5b0d80a6-dd22-41ec-9412-2036b7884cd3)

After uninstalling & reinstalling to version 0.4.8, its GUI no longer appears and the folder it is looking for in the %Temp% directory doesn't exist

**Comment by Tenpi** [comment]
Reinstall the same version 0.4.8 again

**Comment by puff-dayo** [comment]
> After uninstalling & reinstalling to version 0.4.8, its GUI no longer appears and the folder it is looking for in the %Temp% directory doesn't exist

The problem seems to be that although I specified the install path to drive E, the installer writes about 1.5GB of temporary data to drive C and then cleans it. If the C drive does not have enough space for the temporary data, subsequent installations will report an error.

> update to 0.4.8 and tell me if that works

Updated to the latest version and the PDF upscale process seems to be working fine. But the problem now is that the model is reading each page of the PDF (A4 paper size) at 612x792 resolution as input, resulting in a lot of raw information being lost. Hopefully maybe if allowed to manually specify the resolution of the PDF read in would be better.

Thanks a lot. 😊
